packages/ibm-products/src/components/Datagrid/Datagrid/DatagridVirtualBody.tsx

```diff
@@ -85,6 +87,17 @@ const DatagridVirtualBody = (datagridState: DataGridState) => {
       if (headWrapEl) {
         headWrapEl.scrollLeft = virtualBody?.scrollLeft;
       }
+      if (enableSpacerColumn === true) {
```

Comment:
Just a bit more concise this way
```suggestion
      if (enableSpacerColumn) {
```